### DISCUSSION TITLE
bitmap size does not match for 'General001'

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1032] [Developers]
hi all. Im getting this fatal error on opening an organ. I searched discussion and found old post #206. Looks like a problem with size of the bitmap files but i am not able to see where is the problem. I compared other similar files and i see that bitmap sizes are always different and organs load properly. any hint appreciated.

![bitmap size general section](https://user-images.githubusercontent.com/50132661/153748654-55c576b6-1daa-4ed8-b0e5-95ee0de20a95.jpg)
![bitmap size match](https://user-images.githubusercontent.com/50132661/153748664-cc6c257c-28b0-4e6a-885b-d733164837a0.jpg)

**Comment by oleg68** [comment]
@paolo2504 could you post here the text of the error?

**Reply by paolo2504** [reply]
Oleg, the title of this post is exactly the text of the error.

**Comment by willeke1234** [comment]
The sizes of the on and off images must be the same number of horizontal and vertical pixels. The size of the files doesn't matter.

**Reply by paolo2504** [reply]
Thanks Willeke. The fact is that those images seems the same at first sight. Even in other similar samplesets..
Ok I’ll examine images deeply and see the hor and vert pixels. Anyway it seems a very strange error to me..

**Comment by ahall41** [comment]
You need to change the (canvas) size of masque14.png to 1000x750 with GIMP or similar.

**Reply by paolo2504** [reply]
ok thanks i'll try for sure and report. 
Again it seems very strange to me. This particular odf is taken from ODFGrandorgue French site. Many samplesets are very similar, image masque14.png is the same and this is the only organ that returns this fatal error on load...

**Reply by paolo2504** [reply]
> You need to change the (canvas) size of masque14.png to 1000x750 with GIMP or similar.

It works. Many thanks Ahall. I learned something new per your help: in the future i'll check also image dimensions for similar errors!
